gcd/src/test/java/org/spine3/server/storage/datastore/DsAggregateStorageShould.java

```diff
@@ -51,6 +53,24 @@
     }
 
     @BeforeClass
+    public static void setUpAll() {
+        StringifierRegistry.getInstance()
+                           .register(new Stringifier<ProjectId>() {
+                               @Override
+                               protected String toString(ProjectId obj) {
```

Comment:
I don't understand this, as we have JSON-based `Stringifier` for `Message`s, which should be fine for these needs. Please check `core-java`.